server/mdm/android/android.go

```diff
@@ -76,3 +76,5 @@ type MDMAndroidPolicyRequest struct {
 	AppliedPolicyVersion sql.Null[int64]  `db:"applied_policy_version"`
 	PolicyVersion        sql.Null[int64]  `db:"policy_version"`
 }
+
+const APP_STATUS_AVAILABLE = "AVAILABLE"
```

Comment:
Nit, idiomatic Go doesn't use all-caps and underscores, I'd suggest `AppStatusAvailable`.